about.js

```diff
@@ -0,0 +1,68 @@
+const Validate = require('is-my-json-valid')
+const { feedIdRegex, blobIdRegex } = require('ssb-ref')
+
+const { link, stringifyRegex } = require('./util')
+
+function name (userId, name) {
+  return {
+    type: 'about',
+    about: link(userId),
+    name
+  }
+}
+
+function image (userId, imgLink) {
+  return {
+    type: 'about',
+    about: link(userId),
+    image: link(imgLink)
+  }
+}
+
+const schema = {
```

Comment:
i recommend checking out https://spacetelescope.github.io/understanding-json-schema/structuring.html, since this schema could be more DRY if we extracted sub-schemas into definitions and referenced them.